graphql/schema/types/waterfall.graphql

```diff
@@ -24,7 +28,14 @@ type WaterfallBuildVariant {
   builds: [WaterfallBuild!]!
 }
 
+type WaterfallVersion {
+  inactiveVersions: [Version!]
```

Comment:
Since a WaterfallVersion should have either `versions` or `inactiveVersions` populated, I think it's more clear if the other one is clearly represented by `null` (as opposed to `version: null` but `inactiveVersions: []`)